prog/postgres/postgres_resource_nexus.rb

```diff
@@ -90,8 +90,12 @@ def before_run
     nap 5 unless representative_server.vm.strand.label == "wait"
 
     postgres_resource.incr_initial_provisioning
-    register_deadline(:wait, 10 * 60)
-    bud self.class, frame, :trigger_pg_current_xact_id_on_parent if postgres_resource.parent
+    if postgres_resource.parent
+      bud self.class, frame, :trigger_pg_current_xact_id_on_parent
+      register_deadline(:wait, 120 * 60)
```

Comment:
At some point in the future, we could maybe set this adaptively, based on the storage size of the parent?

Reply:
Yep, makes sense.